image/generic/sle16/config.sh

```diff
@@ -64,6 +64,7 @@ systemctl enable suse-migration-product-setup.service
 systemctl enable suse-migration-regenerate-initrd.service
 systemctl enable suse-migration-kernel-load.service
 systemctl enable suse-migration-wicked-networkmanager.service
+systemctl enable suse-migration-pam-config.service
```

Comment:
Can you please also create a `suse-migration-container-pam-config.service` and add this service also to `container/sle16/config.sh`. Thanks

Reply:
Done, thanks for the reminder!